CakePHP/ruleset.xml

```diff
@@ -111,6 +112,7 @@
  <rule ref="Squiz.WhiteSpace.SemicolonSpacing"/>
  <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
  <rule ref="Squiz.WhiteSpace.ScopeClosingBrace"/>
+ <rule ref="Generic.NamingConventions.UpperCaseConstant"/>
```

Comment:
Invalid name, should be `UpperCaseConstantName`

Reply:
Blerg sorry about that.